### ISSUE TITLE
Adjust validation and automatic testing to be location-agnostic

### ISSUE THREAD

**Issue by MObrien-IDM** [fpsim/fpsim#199]
As we parameterize the empowerment module for FPsim-Kenya, we are finding that a lot of the tests fail due to new pars and corresponding data being location-specific. If Senegal doesn't have the data, tests for FPsim-Kenya will fail. Do we need to put in null placeholders in each location par file? That seems burdensome. Let's brainstorm a few ideas here to make this kind of development more straightforward. 

Tagging @cliffckerr @pausz @mzimmermann-IDM @robynstuart and @emilydriano for your thoughts.

**Comment by emilydriano** [comment]
Here are a couple options I see, but curious to hear others' thoughts as well. 
1.	Possibly add some logic to the defaults and/or parameters file that can set some placeholder values for these metrics if the location does not equal Kenya
2.	Override the location parameter in the tests with the location/country in development (in this case, Kenya) - with the thought that long-term either the default location (currently Senegal) would have its parameters/files updated as well, or if the empowerment module (or any similar feature) developed isn't planned to be consistently applied across locations then we should probably create separate tests for these respective locations/countries.
 
I did some testing/tinkering with #2 (overriding the location with 'kenya') and am seeing some initial changes we'd need to make regardless - i.e. getting KeyErrors as 'empowerment', 'age_partnership', and 'urban_prop' are not attributes found in 'pars' /defaults nor in the 'People' object of a sim.

**Comment by MObrien-IDM** [comment]
I think we can also discuss making the default location Kenya now, since it's the most well developed. @emilydriano

**Comment by emilydriano** [comment]
> I think we can also discuss making the default location Kenya now, since it's the most well developed. @emilydriano

Agreed. Let's plan to discuss at the Development meeting next Thurs

**Comment by robynstuart** [comment]
Seems done
